### PR TITLE
Metrics, Logging, Persistent Storage

### PR DESCRIPTION
**Pull request by mglantz**
Hi,

Here's a patch set that adds metrics, logging and persistent storage for both HA and non-HA setups, together with docs for it. Persistent storage is provided via NFS which is setup on MASTER-0. The OpenShift admin user is now automatically made into a cluster administrator and the password for that user is re-used to set a password for Cockpit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,14 +103,18 @@ You should see a folder named '0' and '1'.  In each of these folders, you will s
 
 ## Post-Deployment Operations
 
-This template creates an OpenShift user but does not make it a full OpenShift user.  To do that, please perform the following.
+### Metrics and logging
 
-1. SSH in to master node
-2. Execute the following command:
+To display metrics and logs, you need to logon to OpenShift ( https://publicDNSname:8443 ) go into the logging project, click on the Kubana route and accept the SSL exception in your brower, then do the same with the Hawkster metrics route in the openshift-infra project.
```
Comment (Nordtom) [inline README.md:108]: README.md

**Review by Nordtom — COMMENTED** [review]
Yes

File: README.md (hunk 1 above)
Comment (Nordtom) [reply README.md:108]: deployOpenShift.sh

**Review by haroldwongms — COMMENTED** [review]
looks good

**Pull request merged**